doc/UsersGuide/source/solving.rst

```diff
@@ -158,6 +158,142 @@ with :ref:`SUNDIALS/IDA <sundials_ida>` integrator and with enabled
 automatically by default, when a simulation run is started.
 
 
+.. _initialization :
+
+Initialization
+--------------
+
+To simulate an ODE representation of an Modelica model with one of the methods
+shown in :ref:`cruntime-integration-methods` a valid initial state is needed.
+Equations from an initial equation or initial algorithm block define a desired
+initial system.
+
+Choosing start values
+~~~~~~~~~~~~
+
+Only non-linear iteration variables in non-linear strong components require
```

Comment:
This is true if we are not using tearing. If we use tearing the thing is a bit more involved, because you may have a linear tearing variable that is used to compute a nonlinear variable in the torn section.

Maybe a more precise explanation could be:

Only variables influencing the jacobian of non-linear strong components require start values. These are the the variables appearing in a non-linear way in the residual equations, or influencing torn variables in a non-linear way in the torn equations.

What do you think?

Reply:
We could add that, but in this case _non-linear_ should be equivalent to _influences the jacobian_.

Reply:
I agree that it all boils down to the definition of _non-linear variable_

My point is that this could be ambiguous. In power system models, it is often the case that some variables that only appear in a linear fashion in the equations (hence, they could be defined as _linear variables_) are selected as tearing variables, while some nonlinear variables (which have meaningful start values) are torn out, so those meaningful start values are lost. This is a big deal, because we are not giving start values to these linear variables, but as they influence some nonlinear torn variables, eventually we do influence (badly) the system initialization. 

In fact, currently PowerGrids models only initialize correctly if you activate minimal tearing, precisely for this reason.

Maybe the right solution to this problem is not to change the documentation, but rather to change the tearing algorithm so it avoids this unfortunate situation.